Minor edits to results and discussion sections

daneckaw opened the pull request:
I added small edits to results and discussion sections.

I think that supplementary figure 14 is unnecessary and doesn't bring any new information. If it has to be included in the supplement, I don't think it needs to be refered to in detail in results section - maybe in general as "QC figures". In the current form it can only confuse the reader, because it suggests that we should have some expectations of genome-wide correlation between 5PS and QS (did we expect it to be well correlated or not?). Plus, we're not comparing genome-wide quantification between two methods, but relative differences between reporter constructs.

daneckaw committed:
Minor edits in results.Rmd

daneckaw committed:
Minor edits in discussion.Rmd

ewallace commented:
This is looking good. I will add my edits to @daneckaw's in this branch, then ask @DimmestP to review & merge when done. (not yet)

ewallace committed:
Updated 5PSeq section of results - need to check refs

ewallace committed:
Minor clarifications to materials and methods

Needs to cite nextflow #217 by adding to reference file before it will compile correctly.

ewallace committed:
Bridge paragraph from results to discussion

ewallace commented:
Hi - I have made a couple more edits here, it's looking very good and I would like to send the next cleared-up draft out to Vicent early in the week. Thanks both!

DimmestP commented:
I have added all comments and removed sup figure 14
